transport/awslambda/handler.go

```diff
@@ -0,0 +1,123 @@
+package awslambda
+
+import (
+	"context"
+
+	"github.com/go-kit/kit/endpoint"
+	"github.com/go-kit/kit/log"
+)
+
+// Handler wraps an endpoint.
+type Handler struct {
+	e            endpoint.Endpoint
+	dec          DecodeRequestFunc
+	enc          EncodeResponseFunc
+	before       []HandlerRequestFunc
+	after        []HandlerResponseFunc
+	errorEncoder ErrorEncoder
+	finalizer    []HandlerFinalizerFunc
+	logger       log.Logger
+}
+
+// NewHandler constructs a new handler, which implements
+// the AWS lambda.Handler interface.
+func NewHandler(
+	e endpoint.Endpoint,
+	dec DecodeRequestFunc,
+	enc EncodeResponseFunc,
+	options ...HandlerOption,
+) *Handler {
+	s := &Handler{
```

Comment:
Probably now `h` instead of `s`. Same goes for the methods below.

Reply:
updated e831b16d6ff6e6a903edb7c64e445deebf47b063